### ISSUE TITLE
fontSize setting broken?

### ISSUE THREAD

**Issue by stanidesis** [JoshuaKGoldberg/emojisplosion#8]
Hi, love the work!

I am setting fontSize like so:
```js
const { cancel } = emojisplosions({
      container: $modal[0],
      emojiCount: () => Math.random() * 20 + 20,
      emojis: ["💖", "💕", "💗", "💓", "💝"],
      physics: { fontSize: { max: 54, min: 24 } },
    });
```
And am still seeing the default 14->28 range. Help, I want big emojis!
I love this though...

**Comment by JoshuaKGoldberg** [comment]
Hey, thanks for using this! 💖

That was definitely a bug in the package. It's been fixed in the linked PR and released in the npm package version `1.1.1`.

**Comment by stanidesis** [comment]
Thank you! You rock 💖
